src/engagement_db_to_analysis/analysis_files.py

```diff
@@ -0,0 +1,112 @@
+import csv
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.traced_data.io import TracedDataCSVIO
+from core_data_modules.util import IOUtils
+
+from src.engagement_db_to_analysis.column_view_conversion import analysis_dataset_configs_to_column_configs
+
+
+def export_production_file(traced_data_iterable, analysis_config, export_path):
+    """
+    Exports a column-view TracedData to a production file.
+
+    The production file contains the participant uuid and all the raw_datasets only.
+
+    :param traced_data_iterable: Data to export.
+    :type traced_data_iterable: iterable of core_data_modules.traced_data.TracedData
+    :param analysis_config: Configuration for the export.
+    :type analysis_config: src.engagement_db_to_analysis.configuration.AnalysisConfiguration
+    :param export_path: Path to export the file to.
+    :type export_path: str
+    """
+    IOUtils.ensure_dirs_exist_for_file(export_path)
+    with open(export_path, "w") as f:
+        headers = ["participant_uuid"] + [c.raw_dataset for c in analysis_config.dataset_configurations]
+        TracedDataCSVIO.export_traced_data_iterable_to_csv(traced_data_iterable, f, headers)
+
+
+def _get_analysis_file_headers(column_configs):
+    """
+    Gets the headers for an analysis file.
+
+    The headers are:
+     - "participant_uuid"
+     - Labels for each normal code scheme, in matrix format e.g. "age:25", "s01e01:healthcare".
+     - Raw messages for each dataset.
+
```

Comment:
- Optional special group membership e.g listening_group_member

Reply:
Will add this to the docstring when we add the behaviour